test/utils/shallowEquality.spec.js

```diff
@@ -0,0 +1,71 @@
+import expect from 'expect';
+import shallowEqualScalar from '../../src/utils/shallowEqualScalar';
+import shallowEqual from '../../src/utils/shallowEqual';
+
+describe('Utils', () => {
+  // More info: https://github.com/gaearon/redux/pull/75#issuecomment-111635748
+  describe('shallowEqualScalar', () => {
+    it('should compare the given objects shallowly', () => {
```

Comment:
@gaearon I don't know how to concisely describe shallowEqualScalar. Help please? or just leave it be since I have left the comment?